get_session_request.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+function get_session_request($contents) {
+    $parsedjson = json_decode($contents, true);
+
+    $randomnum = rand(1,9);
+    for($i=0; $i<10; $i++)
+    $randomnum .= rand(0,9);
+
+    $sessionrequest = [
+        '@context' => 'https://irma.app/ld/request/issuance/v2',
+        'credentials' => [[
+            "credential" => 'irma-demo.IRMATube.member',
+            "validity" => strtotime("+6 months"),
+            "attributes" => [
+                "fullname" => $parsedjson['disclosed'][0][0]['rawvalue'],
+                "type" => "premium",
+                "id" => $randomnum
+            ]
+        ]]
+    ];
+
+    return json_encode($sessionrequest);
+}
+
+function error() {
```

Comment:
This function like `stop()` below is not invoked and can be removed.

Reply:
ah, bad copy-pasting. removed now